lua/entities/gmod_wire_expression2/core/vector.lua

```diff
@@ -714,9 +714,8 @@ end
 
 __e2setcost( 5 )
 
---- Gets the vector nicely formatted as a string "[X,Y,Z]"
 e2function string toString(vector v)
-	return ("[%s,%s,%s]"):format(v[1],v[2],v[3])
+	return ("vec(%.2f,%.2f,%.2f)"):format(v[1], v[2], v[3])
```

Comment:
Taking precision away might not be favorable

Reply:
Maybe a second argument to the function for a rounding decimal place?

Reply:
Thought two digits of precision was fine, if they want incredible precision they could just print the individual digits.
Alternatively could use `%g`.

And print already had this output, this is just replacing toString's output with it.